board/safety/safety_hyundai.h

```diff
@@ -252,6 +288,7 @@ static void hyundai_legacy_init(int16_t param) {
   relay_malfunction_reset();
 
   hyundai_legacy = true;
+  hyundai_longitudinal = false;
```

Comment:
why not allow long for the legacy safety mode?

Reply:
addresses could be completely different and I have no way to test, what I have now I know is sufficient at least for my car to not have any errors (which I didn't get on the first try)

Reply:
hyundai legacy is the same as the regular hyndai safety mode, except with a subset of the counter/checksum checks

Reply:
ok, I added it